functional_tests/test_db_online_offline.py

```diff
@@ -18,7 +19,7 @@
 import logging
 log = logging.getLogger(lib.settings.LOGGER)
 
-NUM_ENDPOINTS = 10
+NUM_ENDPOINTS = 13
```

Comment:
I think a better approach would be to have all the API in array or dictionary and then just use count to get the num rather than manually updating global variable.